### PR TITLE
Refactor/register identity

### PR DESCRIPTION
**Pull request by erdoganishe**
Change compile params for hash with 1024 purposes, script updated for 5 sig type

### CODE REVIEW

File: circuits/bigInt/bigIntFunc.circom [hunk 1]
```diff
@@ -430,11 +430,11 @@ function prod_mod(CHUNK_SIZE, CHUNK_NUMBER, A, B, P) {
 // A has CHUNK_NUMBER registers
 // P has CHUNK_NUMBER registers
 // EXP has CHUNK_NUMBER registers
-// CHUNK_NUMBER * CHUNK_SIZE <= 500
+// CHUNK_NUMBER * CHUNK_SIZE <= 513
```
Comment (1KitCat1) [inline circuits/bigInt/bigIntFunc.circom:433]: < 513 ?

File: circuits/identityManagement/circuits/registerIdentityBuilder.circom [hunk 2]
```diff
@@ -68,9 +69,9 @@ template RegisterIdentityBuilder (
 
     // INPUT SIGNALS:
     signal input encapsulatedContent[ENCAPSULATED_CONTENT_SIZE * HASH_BLOCK_SIZE];
-    signal input dg1[DG1_SIZE * HASH_BLOCK_SIZE];
+    signal input dg1[1024];
```
Comment (1KitCat1) [inline circuits/identityManagement/circuits/registerIdentityBuilder.circom:72]: Use predefined constant

File: circuits/identityManagement/circuits/registerIdentityBuilder.circom [hunk 3]
```diff
@@ -68,9 +69,9 @@ template RegisterIdentityBuilder (
...
     signal input dg15[DG15_SIZE * HASH_BLOCK_SIZE];
-    signal input signedAttributes[SIGNED_ATTRIBUTES_SIZE * HASH_BLOCK_SIZE];
+    signal input signedAttributes[1024];
```
Comment (1KitCat1) [inline circuits/identityManagement/circuits/registerIdentityBuilder.circom:74]: Same ^

File: circuits/passportVerification/passportVerificationBuilder.circom [hunk 4]
```diff
@@ -146,9 +140,15 @@ template PassportVerificationBuilder(
...
-
+        // log("Flow result:", passportVerificationFlow[i].flowResult);
+        if (i == 0) {
+            flowResultArray[i] <== passportVerificationFlow[i].flowResult;
+        } else {
+            flowResultArray[i] <== passportVerificationFlow[i].flowResult + flowResultArray[i-1];
+        }
     }
+
+    assert(flowResultArray[FLOW_MATRIX_HEIGHT-1] >= 1);
```
Comment (1KitCat1) [inline circuits/passportVerification/passportVerificationBuilder.circom:151]: Security problem. Use IsGreaterThan(). Assert does not add constraints

File: circuits/identityManagement/circuits/registerIdentityBuilder.circom (hunk 2 above)
Comment (erdoganishe) [reply circuits/identityManagement/circuits/registerIdentityBuilder.circom:72]: done

File: circuits/identityManagement/circuits/registerIdentityBuilder.circom (hunk 3 above)
Comment (erdoganishe) [reply circuits/identityManagement/circuits/registerIdentityBuilder.circom:74]: done

File: circuits/passportVerification/passportVerificationBuilder.circom (hunk 4 above)
Comment (erdoganishe) [reply circuits/passportVerification/passportVerificationBuilder.circom:151]: done

**Pull request merged**